### ISSUE TITLE
Promote sub-group to full group

### ISSUE THREAD

**Issue by poVoq** [AU-Landing-Project/au_subgroups#4]
Would it be possible to add a button in the edit group menu to promote a sub-group to a full, regular group?

**Comment by beck24** [comment]
Yes probably, for now I'm working on upgrading things though.  If you have figured this out yourself in the meantime I'd gladly look merging a PR
